tests/autoyast/installation.pm

```diff
@@ -196,6 +197,11 @@ sub run {
             send_key 'ret';    # press enter if grub timeout is disabled, like we have in reinstall scenarios
             last;              # if see grub, we get to the second stage, as it appears after bios-boot which we may miss
         }
+        elsif (match_has_tag('import-untrusted-gpg-key')) {
```

Comment:
I tried to find any other test suite where we have this popup to be appeared, but did not find.

Do you know why this 'untrusted gpg key' popup is appeared at all in your test?

Reply:
the needle is already in the repo. i cant find any specific match atm but there are some tests that use it